rust/src/builtins/scriptlet_intercept/groupadd.rs

```diff
@@ -13,6 +13,12 @@ use std::os::unix::prelude::PermissionsExt;
 pub(crate) fn entrypoint(args: &[&str]) -> Result<()> {
     fail::fail_point!("intercept_groupadd_ok", |_| Ok(()));
 
+    // Extract the package name from rpm-ostree/bwrap environment.
+    // This also works as a sanity check to ensure we are running in
+    // the context of a scriptlet.
+    let pkgname = std::env::var("RPMOSTREE_SCRIPT_PKG_NAME")
+        .map_err(|_| anyhow!("Could not detect package name for 'groupadd' call."))?;
```

Comment:
Why swallow the error?  How about

```suggestion
    const SCRIPT_PKG_VAR = "RPMOSTREE_SCRIPT_PKG_NAME";
    let pkgname = std::env::var(SCRIPT_PKG_VAR)
        .context("Failed to access {SCRIPT_PKG_VAR} environment variable")?;
```

Reply:
I dropped the error value here because it doesn't bring any additional info/context.
See https://doc.rust-lang.org/stable/std/env/enum.VarError.html.
In this specific usecase of ours, it is effectively equivalent to an "is-present" bool.
That said, even without embedding the original error value, would you prefer a different wording of the error message ? (Although basically it should never occur)

Reply:
If the variable somehow isn't UTF-8, it'd be good to distinguish that from it not being present at all.  For example, the former could happen in the presence of memory corruption somewhere.

Reply:
Ack, changed to that and rebased.